code/datums/components/tackle.dm

```diff
@@ -567,7 +567,7 @@
 			user.visible_message(span_danger("[user] slams head-first into [hit], suffering major cranial trauma!"), span_userdanger("You slam head-first into [hit], and the world explodes around you!"))
 			user.apply_damage(30, BRUTE, spread_damage = TRUE)
 			user.apply_damage(30, STAMINA)
-			user.adjust_confusion(15 SECONDS)
+			user.adjust_confusion(120)
```

Comment:
I will say we should perhaps consider some defines for this

`adjust_confusion(120)` doesn't tell me a lot of what's going on here. But `adjust_confusion(CONFUSION_AMOUNT_ALOT)` is way more understandable. (`#define CONFUSION_AMOUNT_ALOT 120`, `#define CONFUSION_AMOUNT_MIDDLE 60` etc)

Reply:
I think we would need 3 sets of defines in that case.

A weak value for a one of trigger would be something like 25, but for something that adds confusion per second that would be very strong!

 Something like 5 per second would be weak for a per second effect, for effects that pause decay to linearly increase confusion that would be considered strong!

Maybe

```
#def CONFUSION_STRONG 100
#def CONFUSION_MEDIUM 50
#def CONFUSION_WEAK 25

#def CONFUSION_STRONG_PER_SECOND 25
#def CONFUSION_MEDIUM_PER_SECOND 10
#def CONFUSION_WEAK_PER_SECOND 5

#def CONFUSION_STRONG_PER_SECOND_LINEAR 5
#def CONFUSION_MEDIUM_PER_SECOND_LINEAR 2
#def CONFUSION_WEAK_PER_SECOND_LINEAR 1
```

Reply:
What if adjust_confusion was given time as a parameter, and it gives the strength required for it to last that long on its own before it decays completely?

Reply:
That could work too, smart alternative